cmd/kubectl-gadget/main.go

```diff
@@ -19,15 +19,13 @@ import (
 
 	"github.com/spf13/cobra"
 
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/advise"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/audit"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/profile"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/snapshot"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/top"
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/trace"
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/common"
+	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/utils"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/environment"
+	grpcruntime "github.com/inspektor-gadget/inspektor-gadget/pkg/runtime/grpc"
 
-	"github.com/inspektor-gadget/inspektor-gadget/cmd/kubectl-gadget/utils"
+	_ "github.com/inspektor-gadget/inspektor-gadget/pkg/all-gadgets"
```

Comment:
Yeah - right now it would make even more sense for ig to use an `all-gadgets`, as it is not an `all-` in the case of kubectl-gadget, yet. :) So, me need to postpone sharing that for now.